src/itype.h

```diff
@@ -437,12 +437,12 @@ struct islot_gunmod : common_ranged_data {
     /** What kind of weapons can this gunmod be used with (eg. "rifle", "crossbow")? */
     std::set<std::string> usable;
 
-    /** @todo add documentation */
+    /** If this value is set (non-negative), this gunmod functions as a sight. A sight is only usable to aim by a character whose current @ref recoil is at or below this value. */
```

Comment:
This needs to refer to the field in `Character` or `player` class.
I don't think it will detect it if not specified.